src/sage/combinat/designs/covering_array.py

```diff
@@ -0,0 +1,43 @@
+r"""
+Covering Arrays (CA)
+
+A Covering Array, denoted CA`(N,k,v,t)`, is an `n` by `k` array with
+entries from a set of `v` elements with the property that in every
+selection of `t` columns, eachrow contains every sequence of
+`t`-elements at least once.
+
+An Orthogonal Array, denoted OA`(N,k,v,t)` is a covering array with the
+property that each row contains every sequence of `t`-elements exactly
+once
```

Comment:
Missing a dot at the end of the sentence. Also you should add a link to the relevant module dealing with orthogonal arrays here.